ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -482,7 +483,7 @@ export const testClickableButton = (): Promise<boolean> =>
               label: `Don't click me`,
               onClick: () => console.debug(`Why'd you do that?`),
               type: WhisperComponentType.Button,
-              size: ButtonSize.Large,
+              // size: ButtonSize.Large,
```

Comment:
Does this need to be commented out? If we don't need it we should probably just delete it

Reply:
That's probably from me just messing around while writing documentation, and forgetting to clear things. I'll clean that up.